src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/service/cas1/Cas1SpaceBookingService.kt

```diff
@@ -172,7 +174,11 @@ class Cas1SpaceBookingService(
     existingCas1SpaceBooking!!
 
     if (existingCas1SpaceBooking.actualArrivalDateTime != null) {
-      return existingCas1SpaceBooking.id hasConflictError "An arrival is already recorded for this Space Booking"
+      return if (cas1NewArrival.arrivalDateTime.toLocalDate() == LocalDate.ofInstant(existingCas1SpaceBooking.actualArrivalDateTime, ZoneOffset.UTC)) {
```

Comment:
this is a bit of an assumption by me. Do we want to be strcter on the exact second of arrival. This matches on the arrival date only (i.e. the converted `LocalDate`)

Reply:
The date and time is explicitly entered on the UI, so i think this should be a 'to the second' comparison (i.e. a straight comparison of Instants)

Note line 185 and use of `ZoneId`. We're actually using Europe/London as our default zone so we should use ZoneId.systemDefault() where required (although i guess that won't be required anymore in your changed code)

FYI OS timezone is being set in the Dockerfile and JVM is inheriting the OS defaults